Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.18 as build
+FROM golang:1.21 as build
```

Comment:
If you are going to change this, you need to change it in the go.mod file (line 4)

Reply:
updated